tests/utils.go

```diff
@@ -776,6 +776,109 @@ func enableThinpoolMonitoring() {
 	gomega.Expect(err).To(gomega.BeNil(), "run lvchange command")
 }
 
+// This creates loopdevice using the size passed as arg,
+// Uses the new loop device to create PV. returns loopdevice name to the caller.
+func createPV(size int) string {
+	fmt.Printf("Creating device\n")
+	t := time.Now().Format("20060102150405")
+	filename := fmt.Sprintf("/tmp/openebs_ci_disk_%s.img", t)
+
+	size_str := strconv.Itoa(size) + "G"
+	args_file := []string{
+		"truncate", "-s",
+		size_str, filename,
+	}
+	_, _, err := execAtLocal("sudo", nil, args_file...)
+	gomega.Expect(err).ShouldNot(gomega.HaveOccurred(), "create device failed")
+	args_loop := []string{
+		"losetup", "-f",
+		filename, "--show",
+	}
+
+	stdout_loop, _, err := execAtLocal("sudo", nil, args_loop...)
```

Comment:
We can run the test using sudo, e.g.: https://github.com/openebs/dynamic-localpv-provisioner/blob/v3.5.x/Makefile#L146

If the test is using a shell script, e.g. test.sh, run it using `sudo bash -c "./test.sh"`

We should not call sudo implicitly like so.

Reply:
The main ci script in lvm repo runs as non-root user in github actions. Most of the work done inside that script is doing `sudo` today e.g. `sudo vgcreate`. These new wrappers seem to be emulating same.

Reply:
This feels like tech debt. But oh well.

Reply:
We could have a wrapper for calls which require sudo. In general IMHO it's better to use sudo only for operations which need it rather than blindly run everything with sudo as that seems a lot more dangerous.